composer.json

```diff
@@ -7,6 +7,9 @@
         "AFL-3.0"
     ],
     "type": "maho-source",
+    "repositories": [
+        { "type": "vcs", "url": "https://github.com/justinbeaty/maho-phpstan-plugin.git" }
```

Comment:
Just a note to make sure we remember to change this.

Reply:
maho-phpstan-plugin 1.2.0 is released ;-)